ui/src/ScholarReader.tsx

```diff
@@ -308,6 +310,9 @@ class ScholarReader extends React.PureComponent<ScholarReaderProps, State> {
     eventBus.on("documentloaded", (eventData: DocumentLoadedEvent) => {
       this.setState({ pdfDocument: eventData.source });
```

Comment:
I'm not sure why, but this event also appears to fire on document load (or at least PDF load) hahaha

Reply:
The `sidebarchanged` event?

Can you find evidence in the `pdf.js` code that the event will always fire before `documentloaded` is called (and that "before" is important, because it's once the document is loaded that all of our annotation layers will be loaded)? If so, just make a note of it in the code and we'll consider this one resolved.